code/public/index.html

```diff
@@ -4,16 +4,11 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+    <meta property="og:title" content="Technigo React To do list by Irupé Pozo Graviz"/>
+        <meta property="og:description" content="A website displaying a todo list."/>
+        <meta property="og:image" content="/landing_page.png"/>
```

Comment:
Nice adding og-tags! You can check what they look like here if you haven't already:
https://www.opengraph.xyz/url/https%3A%2F%2Firupes-to-do-app.netlify.app%2F